docs/content/2.components/table.md

```diff
@@ -29,6 +29,7 @@ Use the `columns` prop to configure which columns to display. It's an array of o
 - `sortable` - Whether the column is sortable. Defaults to `false`.
 - `direction` - The sort direction to use on first click. Defaults to `asc`.
 - `class` - The class to apply to the column cells.
```

Comment:
Why not use the `class` field already dedicated to this?

Reply:
The `class` field only seems to be applied to the `<th>` of the column, if I use `block lg:hidden` there it will only hide the header and not the data cells `<td>`.
I would prefer having a way to add a class from the columns list instead of having to modify the `rows` I pass into the Table component to set classes on data cells.